packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -92,14 +92,19 @@ export const FileUploadButton = (props: FileUploadButtonProps): JSX.Element => {
  * It will return `<></>` for stable builds.
  * @internal
  */
-export const FileUploadButtonWrapper = (): JSX.Element => {
-  return <div>{FileUploadConditionalCode()}</div>;
-};
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const FileUploadConditionalCode = (): any => {
+export const FileUploadButtonWrapper = (
+  // To make conditional compilation not throw errors.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  props: Pick<FileUploadButtonProps, 'accept' | 'multiple' | 'fileUploadHandler'>
```

Comment:
You won't have to do this if you 

(1) define the type that contains file upload options (accept, multiple, handler etc) - unconditionally
(2) Use that type in this function -- unconditionally
(3) Only for beta -- add a field to ChatComposite that uses that type && export that type from index.ts

I think trying to keep the conditional compilation as close as possible the API surface is helpful -- it reduces how much conditional compilation you need, and that means that beta/stable are effectively closer together (so lower risk when features go from beta to stable)

Reply:
I think you misunderstand this comment. Stable build throws an unused var error because props is no longer used in stable. To disabled that error, I added the comment // eslint-disable-next-line @typescript-eslint/no-unused-vars